src/QuickMarcEditor/QuickMarcEditWrapper.test.js

```diff
@@ -8,7 +8,8 @@ import {
 } from '@testing-library/react';
 import faker from 'faker';
 import noop from 'lodash/noop';
-import { useLocation } from 'react-router';
```

Comment:
Could you please fix eslint error ('screen' is defined but never used. Allowed unused vars must match /React/u 'screen' is defined but never used. Allowed unused vars must match /React/u)?